syft/frameworks/torch/he/fv/evaluator.py

```diff
@@ -84,4 +89,79 @@ def _add_plain_plain(self, pt1, pt2):
                 arguments.
         """
         pt1, pt2 = copy.deepcopy(pt1), copy.deepcopy(pt2)
-        return PlainText(poly_add_mod(pt1.data, pt2.data, self.plain_modulus))
+        return PlainText(poly_add_mod(pt1.data, pt2.data, self.plain_modulus, self.poly_modulus))
+
+    def _mul_cipher_cipher(self, ct1, ct2):
+        ct1, ct2 = ct1.data, ct2.data
+
+        if len(ct1) > 2:
+            # TODO: perfrom relinearisation operation.
+            raise RuntimeError(
+                "Cannot multiply ciphertext of size >2, Perform relinearisation operation."
+            )
+        if len(ct2) > 2:
+            # TODO: perfrom relinearisation operation.
+            raise RuntimeError(
+                "Cannot multiply ciphertext of size >2, Perform relinearisation operation."
+            )
+
+        ct10, ct11 = copy.deepcopy(ct1)
+        ct20, ct21 = copy.deepcopy(ct2)
+
+        # lenght of coeff_modulus determine no of polynomials.
+        result = [
+            [0] * len(self.coeff_modulus),
+            [0] * len(self.coeff_modulus),
+            [0] * len(self.coeff_modulus),
+        ]
+
+        # coefficients raised by q(coefficient modulus)
+        for i in range(len(self.coeff_modulus)):
+            ct10[i] = [x * self.coeff_modulus[i] for x in ct10[i]]
+            ct11[i] = [x * self.coeff_modulus[i] for x in ct11[i]]
+            ct20[i] = [x * self.coeff_modulus[i] for x in ct20[i]]
+            ct21[i] = [x * self.coeff_modulus[i] for x in ct21[i]]
+
+        poly_mod = np.array([1] + [0] * (self.poly_modulus - 1) + [1])
```

Comment:
You might transform this poly_mod creation into a function for reuse

Reply:
Actually, it was not working so I put everything in one place to check what is causing the problem. It is more like a rough work right now, will update everything in a nice way once things start working.